Paintroid/src/main/java/org/catrobat/paintroid/listener/BottomBarScrollListener.kt

```diff
@@ -0,0 +1,95 @@
+/**
+ * Paintroid: An image manipulation application for Android.
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http:></http:>//developer.catrobat.org/credits>)
+ *
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http:></http:>//www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.listener
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.HorizontalScrollView
+
+class BottomBarScrollListener : HorizontalScrollView {
+    private lateinit var previous: View
+    private lateinit var next: View
+
+    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(
+        context,
+        attrs,
+        defStyle
+    ) {
```

Comment:
The curly braces seem to be redunant, so it would probably be quite nice, if you could remove them :)